web/app/js/components/ServiceMesh.jsx

```diff
@@ -249,26 +253,37 @@ export default class ServiceMesh extends React.Component {
   }
 
   renderAddDeploymentsMessage() {
-    return (
-      <div className="mesh-message">
-        {this.unaddedDeploymentCount() === 0 ?
+    let instructions = (<div className="instructions">Add one or more deployments to the deployment.yml file <br /><br />
```

Comment:
this doesn't have to be redefined every render. Let's move it out into a const at the top of the file.

better yet, since this text is also used in the CallToAction, place it in a util file (or somewhere) and reuse the code.

Reply:
good point, did that! (also will submit a follow up commit using the exported const in CallToAction in a sec)